src/Error/Debugger.php

```diff
@@ -32,6 +33,16 @@
  */
 class Debugger
 {
+    use InstanceConfigTrait;
```

Comment:
~~Any reason to not use `StaticConfigTrait`? While Debugger contains a singleton it probably shouldn't long term.~~ I'm a tool ignore this.